pandas/io/gbq.py

```diff
@@ -29,9 +29,11 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
     The main method a user calls to execute a Query in Google BigQuery
     and read results into a pandas DataFrame.
 
-    Google BigQuery API Client Library v2 for Python is used.
-    Documentation is available `here
-    <https://developers.google.com/api-client-library/python/apis/bigquery/v2>`__
+    This function requires the `pandas-gbq package
```

Comment:
So need to have a section in the pandas-gbq docs on how to deal with < 0.3.0 (IOW list the deps and such).

Also I would remove the referenes here tot the google cloud library itself, you just need to install pandas-gbq, which depending the installed version has different requirements.

also there is a referene in install.rst to pandas-gbq, see if it needs updating.

Reply:
Done, done, and done.